content/blog/teachopencadd-kinase.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "TeachOpenCADD Kinase edition is out!"
+date: 2021-12-10T00:00:00+02:00
+author: Talia B. Kimber & Dominique Sydow
+---
+
+## Assessing kinase similarity, or how to avoid side effects?
+
+Kinases are highly conserved in their binding site, which presents a challenge since one ligand may bind not only the designated target (on-target) but also other targets (off-targets), causing mild to severe sides effects. Being able to assess kinase similarity could therefore give insight into potential side-effects.
+
+The proposed pipeline, part of the TeachOpenCADD project, allows the study of kinase similarities from four different angles in an automated and modular fashion.
+
+A short kinase introduction, as well as an overall conclusion, are part of the special kinase edition, establishing 6 news notebooks!
+The analysis of kinase similarity uses information from:
+
+-  sequence,
+-  structure,
```

Comment:
```suggestion
-  pocket sequence,
-  pocket structure,
```

Reply:
_Optional_ addition: Adding the resources we are using (KLIFS, KiSSim, ChEMBL). Could go in parentheses at the end of each bullet point or as a stand-alone sentence.

Reply:
Yes, good idea! Will have the resources in parentheses!

Reply:
Good idea, resources including links could be added.